packages/picasso/src/Popper/Popper.tsx

```diff
@@ -131,14 +131,19 @@ export const Popper = forwardRef<PopperJs, Props>(function Popper (props, ref) {
     }
   }, [isCompactLayout, open])
 
+  const mergedPopperOptions = React.useMemo(
```

Comment:
why merged? Maybe `memoed`? 
```suggestion
  const memoedPopperOptions = React.useMemo(
```